src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -20,10 +20,14 @@ package object rlp {
 
   def encode[T](input: T)(implicit enc: RLPEncoder[T]): Array[Byte] = RLP.encode(enc.encode(input))
 
+  def encodeToRlp[T](input: T)(implicit enc: RLPEncoder[T]): RLPEncodeable = enc.encode(input)
+
   def encode(input: RLPEncodeable): Array[Byte] = RLP.encode(input)
 
   def decode[T](data: Array[Byte])(implicit dec: RLPDecoder[T]): T = dec.decode(RLP.rawDecode(data))
 
+  def decode[T](data: RLPEncodeable)(implicit dec: RLPDecoder[T]): T = dec.decode(data)
```

Comment:
Maybe something like `toRLPEncodeable` and `fromRLPEncodeable` to make it more explicit that we are transforming objects to and from `RLPEncodeable`. Either way, we might we able to not have these functions (or at least `encodeToRLP`) using what @AlanVerbner mentioned.